packages/kbn-repo-packages/modern/parse_package_manifest.js

```diff
@@ -103,6 +104,14 @@ function validatePackageManifestPlugin(plugin, repoRoot, path) {
     );
   }
 
+  if (runtimePluginDependencies !== undefined && !isArrOfIds(runtimePluginDependencies)) {
+    throw err(
+      `plugin.runtimePluginDependencies`,
+      optionalPlugins,
```

Comment:
```suggestion
      runtimePluginDependencies,
```

Reply:
Thanks, fixed in https://github.com/elastic/kibana/pull/167113/commits/ae863ae608bc7c1db2440a8be1ed3028a35eb452